datasets/nrcan-landcover/nrcan-landcover-example.ipynb

```diff
@@ -0,0 +1,1352 @@
+       "| 0.00,-30.00, 586590.00|\n",
+       "| 0.00, 0.00, 1.00|</dd><dt><span>resolution :</span></dt><dd>30.0</dd></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.DataArray 'stackstac-94f5f71e7b85a6b5b00d5c297a8b05a1' (time: 2, band: 1, y: 6002, x: 6855)>\n",
+       "dask.array<fetch_raster_window, shape=(2, 1, 6002, 6855), dtype=uint8, chunksize=(1, 1, 1024, 1024), chunktype=numpy.ndarray>\n",
+       "Coordinates: (12/21)\n",
+       "  * time               (time) datetime64[ns] 2015-01-01 2015-01-01\n",
+       "    id                 (time) <U21 'CAN_LC_2015_CAL_12_03' 'CAN_LC_2015_CAL_1...\n",
+       "  * band               (band) <U9 'landcover'\n",
+       "  * x                  (x) float64 -2.058e+06 -2.058e+06 ... -1.852e+06\n",
+       "  * y                  (y) float64 5.866e+05 5.866e+05 ... 4.066e+05 4.066e+05\n",
+       "    proj:epsg          int64 3978\n",
+       "    ...                 ...\n",
+       "    title              <U24 'Land cover of Canada COG'\n",
+       "    label:type         <U6 'raster'\n",
+       "    end_datetime       <U20 '2020-01-01T00:00:00Z'\n",
+       "    sci:doi            <U14 '10.4095/315659'\n",
+       "    raster:bands       object {'nodata': 0, 'sampling': 'area', 'data_type': ...\n",
+       "    epsg               int64 3978\n",
+       "Attributes:\n",
+       "    spec:        RasterSpec(epsg=3978, bounds=(-2057730.0, 406530.0, -1852080...\n",
+       "    crs:         epsg:3978\n",
+       "    transform:   | 30.00, 0.00,-2057730.00|\\n| 0.00,-30.00, 586590.00|\\n| 0.0...\n",
+       "    resolution:  30.0"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# The STAC metadata contains some information we'll want to use when creating\n",
+    "# our merged dataset. Get the EPSG code of the first item and the nodata value.\n",
+    "item = next(search.get_items())\n",
+    "epsg = proj.ext(item).epsg\n",
+    "nodata = raster.ext(item.assets[\"landcover\"]).bands[0].nodata\n",
+    "items = [pc.sign(item).to_dict() for item in search.get_items()]\n",
+    "\n",
+    "# Create a single DataArray from out multiple resutls with the corresponding\n",
+    "# rasters projected to a single CRS. Note that we set the dtype to ubyte, which\n",
+    "# matches our data, since stackstac will use float64 by default.\n",
+    "stack = stackstac.stack(\n",
+    "    items, epsg=epsg, dtype=np.ubyte, fill_value=nodata, bounds_latlon=bounds_latlon\n",
+    ")\n",
+    "\n",
+    "stack"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5dd4be3e",
+   "metadata": {},
+   "source": [
+    "Start up a local Dask cluster to allow us to do parallel reads. Use the following URL to open a dashboard in the Hub's Dask Extension. "
```

Comment:
I could go either way on this, but it's probably fine to just remove the Dask cluster (it'll still use Dask in the background to read in parallel with a thread pool).

If you do want to use a cluster, then can you make sure to handle the warning about port 8787 being in use? You probably have another Dask cluster running (either in this notebook or another one).